ts/Extensions/RangeSelector.ts

```diff
@@ -950,6 +950,12 @@ class RangeSelector {
                 return;
             }
         } else if (type === 'all' && baseAxis) {
+            // If the navigator exist and the axis range is declared reset that
+            // range and from now on only use the range set by a user, #14742.
+            if (chart.navigator) {
+                chart.navigator.baseSeries[0].xAxis.options.range = void 0 as any;
```

Comment:
Please, check if `baseSeries[]` is defined. I know, it might not be a real use-case, but it's better to not introduce this bug: https://jsfiddle.net/BlackLabel/4nq0ca2b/ (click on "All" button and observe console)